physionet-django/project/views.py

```diff
@@ -51,13 +51,14 @@
     UploadedDocument,
 )
 from project.projectfiles import ProjectFiles
-from project.validators import validate_filename
+from project.validators import validate_filename, validate_gcs_bucket_object
 from user.forms import AssociatedEmailChoiceForm
 from user.models import CloudInformation, CredentialApplication, LegacyCredential, User, Training
 
 from django.db.models import F, DateTimeField, ExpressionWrapper
 
 LOGGER = logging.getLogger(__name__)
+MAX_GCS_OBJECT_NAME_LENGTH = 256
```

Comment:
`MAX_GCS_OBJECT_NAME_LENGTH` is also defined in physionet-django/project/validators.py. If it is needed here, it could be imported (though I don't see it being used).

Reply:
Ah, i missed that. just pushed the change removing that.